app/components/contact_details_component.rb

```diff
@@ -0,0 +1,21 @@
+class ContactDetailsComponent < ActionView::Component::Base
+  def initialize(contact_details_form:)
+    @contact_details_form = contact_details_form
+    @contact_details_form_rows = convert_to_rows
+  end
+
+private
+
+  def convert_to_rows
+    [phone_number_row]
+  end
+
+  def phone_number_row
+    {
+      key: I18n.t('application_form.contact_details.phone_number.label'),
```

Comment:
```suggestion
      key: t('application_form.contact_details.phone_number.label'),
```

Not necessary in `ActionView::Component` :)